src/plugins/gauge/components/Gauge.vue

```diff
@@ -299,6 +397,7 @@ export default {
         this.openmct.time.off('timeSystem', this.setTimeSystem);
     },
     methods: {
+        getLimitDegree: getLimitDegree,
```

Comment:
Not a big deal at all but I think you could just have `getLimitDegree,` here, if you like.

Reply:
🤔